presto-docs/src/main/sphinx/release/release-0.224.rst

```diff
@@ -0,0 +1,38 @@
+=============
+Release 0.224
+=============
+
+General Changes
+---------------
+
+* Fix mis-swapped CPU time and scheduled time in query plan.
+* Fix correctness bug on queries containing nested coercible type cast since release 0.220.
+* Add support for recoverable grouped execution for writing unpartitioned table.
+* Add support for index-based access to row-type field with subscript.
+
+Security Changes
+----------------
+
+* Fix security issue with nextUri protocol
```

Comment:
Add a period (.)at the end of the sentence.

Reply:
```
Fix a security issue
```

Reply:
Need to expand what is ``nextUri protocol``.

Reply:
cc @rschlussel

Reply:
What about 
`Fix a security issue in the client protocol`? 

The nextUri protocol is the protocol for telling clients where they can get their next results from.

Reply:
I'm not an expert on release note, but `client protocol` seems to be at a higher level than `nextUri` protocol. Do we need to expand on the impact of the security issue (what would happen if not fixed)?

Reply:
so let's go with client protocol then.  I don't think we need to elaborate on the security issue.  Not sure though.